addons/binding/org.openhab.binding.homie/build.properties

```diff
@@ -0,0 +1,8 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/,\
+               lib/org.eclipse.paho.client.mqttv3-1.1.0.jar,\
```

Comment:
Can you add an about.html and make sure it is referred to from this file.

[Coding guidelines B.2](http://docs.openhab.org/developers/development/guidelines.html#b-osgi-bundles)